libraries/AP_Airspeed/AP_Airspeed_DLVR.cpp

```diff
@@ -104,18 +104,32 @@ void AP_Airspeed_DLVR::timer()
 
     if ((data >> STATUS_SHIFT)) {
         // anything other then 00 in the status bits is an error
-        Debug("DLVR: Bad status read %d", data >> STATUS_SHIFT);
+        Debug("DLVR: Bad status read %d", (int)(data >> STATUS_SHIFT));
```

Comment:
Shouldn't this be unsigned (and the format fixed)?

Reply:
Yes, I may have just done the minimal amount to get it to compile by casting to int. It doesn't really matter since there's only 2 bits of data here (and arguably should just be a hex format), but it should be correct now.